test/benchmarks/driverBench/index.js

```diff
@@ -47,7 +47,7 @@ benchmarkRunner
     const multiBench = average(Object.values(microBench.multiBench));
 
     const parallelBench = average([
-      microBench.parallel.ldjsonMultiFileUpload,
+      microBench.parallel.ldjsonMultiFileUpload ?? 0,
```

Comment:
I worry that setting it to 0 would mess up the tracked result, could we just take it out? Or set it to the currently known average value if we're concerned about preserving the parallel measurement history?

Reply:
I agree, removing it makes more sense. I'd rather have accurate numbers than simulate a value here if that sounds good.